packages/plans-grid/src/plans-interval-toggle/index.tsx

```diff
@@ -57,20 +57,24 @@ const PlansIntervalToggle: React.FunctionComponent< PlansIntervalToggleProps > =
 	maxMonthlyDiscountPercentage,
 	className = '',
 } ) => {
-	const { __, hasTranslation } = useI18n();
+	const { __, _x, hasTranslation } = useI18n();
 	const locale = useLocale();
 
 	const fallbackMonthlyLabel = __( 'Pay monthly', __i18n_text_domain__ );
 	// Translators: intended as "pay monthly", as opposed to "pay annually"
-	const newMonthlyLabel = __( 'Monthly', __i18n_text_domain__ );
+	const newMonthlyLabel = _x( 'Monthly', 'Adverb (as in "Pay monthly")', __i18n_text_domain__ );
 	const monthlyLabel =
-		locale === 'en' || hasTranslation?.( 'Monthly' ) ? newMonthlyLabel : fallbackMonthlyLabel;
+		locale === 'en' || hasTranslation?.( 'Monthly', 'Adverb (as in "Pay monthly")' )
+			? newMonthlyLabel
+			: fallbackMonthlyLabel;
 
 	const fallbackAnnuallyLabel = __( 'Pay annually', __i18n_text_domain__ );
 	// Translators: intended as "pay annually", as opposed to "pay monthly"
-	const newAnnuallyLabel = __( 'Annually', __i18n_text_domain__ );
+	const newAnnuallyLabel = _x( 'Annually', 'Adverb (as in "Pay annually")', __i18n_text_domain__ );
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'renews annually', { context: 'Informative text for renewals in /checkout'} )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/plans-monthly-toggle-translations-context)